pkg/descheduler/evictions/evictions.go

```diff
@@ -90,11 +90,11 @@ func (pe *PodEvictor) EvictPod(pod *v1.Pod, node *v1.Node) (success bool, err er
 	success, err = EvictPod(pe.client, pod, pe.policyGroupVersion, pe.dryRun)
 	if success {
```

Comment:
Based on the current implementation of `EvictPod`, when `success` is `true`, `err` is always `nil`, when `success` is `false`, `err` is never `nil`. @seanmalloy since you started, can you remove the `bool` return argument from EvictPod and simply return just the error?

Reply:
Fixed in commit 7039b6c8aafbad333fa21b3604ba1dae0a7f9b15